src/System.Private.ServiceModel/tools/test/BridgeCertificateInstaller/CertificateInstaller.cs

```diff
@@ -64,66 +66,64 @@ public static int Main(string[] args)
 
     public static bool GetCertificate(out byte[] certificateBytes)
     {
-        string endpoint = Endpoints.Https_DefaultBinding_Address;
-        string bridgePort = TestProperties.GetProperty(TestProperties.BridgePort_PropertyName);
-
         certificateBytes = default(byte[]);
-
-        Uri uri;
+        string address = ServiceUtilHelper.ServiceUtil_Address;
+        Console.WriteLine("  Retrieving certificate from:" + address);
+        ChannelFactory<IUtil> factory = null;
+        IUtil serviceProxy = null;
+        BasicHttpBinding basicHttpBinding = new BasicHttpBinding();
         try
         {
-            if (Uri.TryCreate(endpoint, UriKind.RelativeOrAbsolute, out uri))
-            {
-                string certAsPemUri = string.Format(
-                    "http://{0}:{1}/resource/WcfService.CertificateResources.CertificateAuthorityResource?exportAsPem=true", uri.Host, bridgePort);
-
-                Console.WriteLine("  Bridge hostname is: '{0}'", uri.Host);
-                Console.WriteLine("  Retrieving certificate from:");
-
-                Console.WriteLine(certAsPemUri);
-                Console.WriteLine();
-
-                HttpClient client = new HttpClient();
-                var response = client.GetAsync(certAsPemUri).GetAwaiter().GetResult();
+            factory = new ChannelFactory<IUtil>(basicHttpBinding, new EndpointAddress(address));
+            serviceProxy = factory.CreateChannel();
+            certificateBytes = serviceProxy.GetRootCert(true);
+            Console.WriteLine("    ... read {0} bytes from Bridge", certificateBytes.Length);
```

Comment:
I'm going to change it to "test service" but we need to come up with a clean consistent name.